cl/phase1/network/services/batch_signature_verification.go

```diff
@@ -0,0 +1,158 @@
+package services
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/Giulio2002/bls"
+	sentinel "github.com/erigontech/erigon-lib/gointerfaces/sentinelproto"
+	"github.com/erigontech/erigon-lib/log/v3"
+)
+
+const (
+	BatchSignatureVerificationThreshold = 100
+)
+
+var (
+	batchCheckInterval          = 50 * time.Millisecond
```

Comment:
feel free to increase this to 500 milliseconds. i did some research and it should be fine anyway